src/templates/renew-membership-page.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from "react"
+import { connect } from 'react-redux'
+import { navigate } from "gatsby";
+import Layout from '../components/Layout'
+import TopBar from "../components/TopBar";
+import NavbarV2 from "../components/NavbarV2";
+import Header from "../components/Header";
+import SEO from "../components/SEO";
+import 'openstack-uicore-foundation/lib/css/components/index.css';
+import { AjaxLoader } from "openstack-uicore-foundation/lib/components";
+import { MEMBERSHIP_TYPE_INDIVIDUAL, renewMembership } from "../actions/user-actions";
+
+import leftArrow from '../img/svg/arrow-left.svg'
+
+import '../style/modules/_renew-membership.scss';
+import LinkComponent from "../components/LinkComponent";
+
+export const RenewMembershipPageTemplate = ({
+  isLoggedUser,
+  location,
+  loading,
+  renewMembership,
+  userProfile
+}) => {
+
+  const [acceptedTerms, setAcceptedTerms] = useState(false);
+
+  useEffect(() => {
+    // if (userProfile.membership_type === "Individual") navigate("/a/profile")
+  }, [])
+
+  return (
+    <div>
+      <div className="wrapper project-background">
+        <TopBar />
+        <NavbarV2 isLoggedUser={isLoggedUser} />
+      </div>
+      <AjaxLoader show={loading} size={120} />
+      <main className="main">
+        <div className="content">
+          <section className="section about-s1-main renew-section">
+            <div className="container about-s1-container" style={{ position: "relative", overflow: "visible" }}>
+              <div className="columns">
+                <div className="column is-three-fifths text-column">
+                  <h1>
+                    <span className="renew-membership-title">RENEW YOUR OPENINFRA MEMBERSHIP</span></h1>
+                  {userProfile.membership_type !== MEMBERSHIP_TYPE_INDIVIDUAL ?
```

Comment:
@tomrndom we need to add a state prop that is initialized on false acceptedRenewal or similar
and check on that not on userProfile.membership_type
that state var will be set to true after the user click on the action